src/hooks/usePopularCities.js

```diff
@@ -1,18 +1,18 @@
 import { useMemo } from 'react';
 import { useQuery } from '@tanstack/react-query';
-import fetchPopularCities from '../services/PopularCitiesServise';
+import fetchCities from '../services/CitiesServise';
 import transformCityItem from '../data/transformCityItem';
 
 const POPULAR_CITIES_QUERY_KEY = 'popularCities';
 
 const usePopularCities = () => {
   const { data, error, isLoading } = useQuery({
     queryKey: [POPULAR_CITIES_QUERY_KEY],
-    queryFn: fetchPopularCities,
+    queryFn: fetchCities,
```

Comment:
We can add staleTime param here because the list of popular cities won't be updated so often

```suggestion
   staleTime: 1000 * 60 * 2,
```